src/Draco.FormatterEngine/FormatterSettings.cs

```diff
@@ -22,10 +22,7 @@ internal sealed class FormatterSettings
     /// </summary>
     public string Indentation { get; init; } = "    ";
 
-    /// <summary>
-    /// True, if newlines in strings should be normalized to the <see cref="Newline"/> sequence.
-    /// </summary>
-    public bool NormalizeStringNewline { get; init; } = true;
+    public int LineWidth { get; init; } = 160;
 
     public string IndentationString(int amount = 1)
```

Comment:
Make this internal

Reply:
That was code from the previous formatter, i'll remove it.